includes/wikia/nirvana/WikiaParserTagController.class.php

```diff
@@ -69,7 +69,7 @@ protected function registerResourceLoaderModules( Parser $parser ) {
 		return $markerId;
 	}
 
-	public final function onParserAfterTidy( Parser &$parser, &$text ) {
+	public final function onParserAfterTidy( Parser $parser, &$text ) {
```

Comment:
Let's try to find such places automatically, `grep` to the rescue? Or maybe update the `stylize.php` from the upstream to clean-up such cases?